packages/cli/src/commands/test.ts

```diff
@@ -164,45 +163,51 @@ export default class Test extends Command {
         hasFailures: true,
         runError: message,
       })
-      process.exitCode = 1
+      throw new Error(`Scheduler failure:, ${message}.`)
```

Comment:
I think that we shouldn't throw an error in this case. Other checks are still running as part of `checkly test` - I think that we should let them all finish.